src/core/execute.h

```diff
@@ -253,7 +253,7 @@ struct ExecContext {
         char *apparmor_profile;
         char *smack_process_label;
 
-        char **read_write_paths, **read_only_paths, **inaccessible_paths, **exec_paths, **no_exec_paths;
+        char **read_write_paths, **read_only_paths, **inaccessible_paths, **exec_paths, **no_exec_paths, **exec_search_paths;
```

Comment:
please move this elsewhere... the search paths and the other lists here are very different concepts. i.e. all the others are about fs namespacing. It's confusing putting them here in the same line, since that suggests they are somehow related, but they really aren't